### DISCUSSION TITLE
Can't find get_router_query_engine

### DISCUSSION THREAD

**Discussion by Jalam1001** [run-llama/llama_index#14558] [General]
Hi trying to use . where can I find get_router_query_engine

from utils import get_router_query_engine

query_engine = get_router_query_engine("metagpt.pdf")

get this error ModuleNotFoundError: No module named 'utils'

--------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-59-16329dcd6c95>](https://localhost:8080/#) in <cell line: 1>()
----> 1 from utils import get_router_query_engine
      2 
      3 query_engine = get_router_query_engine("metagpt.pdf")

ModuleNotFoundError: No module named 'utils'

thanks

**Comment by Mayurpatel36** [comment]
https://github.com/sulaiman-shamasna/Agentic-RAG-with-LlamaIndex/blob/main/router_query_engine/utils.py


Donwload the utils.py file from this repo and keep it in your repository. It isn't calling a built in library its calling the python file utils
